### DISCUSSION TITLE
Not getting correct error message while hitting contract method

### DISCUSSION THREAD

**Discussion by harshitaRajput18** [WalletConnect/walletconnect-monorepo#5253] [Developer Support]
When I use    const { connector } = useAccount(); let provider = await connector?.getProvider(); this method to get provider and use this provider to hit the contract method then I'm not getting exact error. It always says Internal Json Rpc error. but when I hard code the RPC then it works fine. Please suggest me the solution.

**Comment by therealjosephdev** [comment]
I need to review the code. Could invite me to the repository
